### ISSUE TITLE
upload_analytic method raises a NameError

### ISSUE THREAD

**Issue by mattphotonman** [voxel51/api-py#38]
```python
from voxel51.api import API
api = API()
api.upload_analytic("/path/to/analytic.json")

---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-4-1f6d68783277> in <module>
----> 1 api.upload_analytic("/path/to/analytic.json")

~/api-py/voxel51/api.py in upload_analytic(self, doc_json_path)
    162         '''
    163         endpoint = self.base_url + "analytics"
--> 164         filename = os.path.basename(path)
    165         mime_type = _get_mime_type(doc_json_path)
    166         with open(doc_json_path, "rb") as df:

NameError: name 'path' is not defined
```

**Comment by hoddr** [comment]
@mattphotonman `"path/to/analytic.json"` must be filled in with the actual path to the file in your file system. That is a placeholder/template example!

**Comment by hoddr** [comment]
Ahh, but that stack trace may suggest there is another issue too

**Comment by hoddr** [comment]
Addressed in PR #39. Thanks @mattphotonman!
